src/push_api_clientpy/platformclient.py

```diff
@@ -187,10 +188,10 @@ def pushFileContainerContent(self, sourceId: str, fileContainer: FileContainer):
         return self.session.put(url=url, params=queryParams, headers=self.__headers())
 
     def __basePushURL(self):
-        return f'https://api.cloud.coveo.com/push/v1/organizations/{self.organizationid}'
+        return f'https://api-{self.region}.cloud.coveo.com/push/v1/organizations/{self.organizationid}'
```

Comment:
I don't think `https://api-us.cloud.coveo.com/push/v1/organizations/` is equal to `https://api.cloud.coveo.com/push/v1/organizations/`.

I can't find mentions of `-us`
https://docs.coveo.com/en/2976/manage-an-organization/deployment-regions-and-strategies